proxy/certs/certs.go

```diff
@@ -185,6 +204,27 @@ func parseCert(pemCert string) (*x509.Certificate, error) {
 	return x509.ParseCertificate(bl.Bytes)
 }
 
+// Find the first matching IP address by user input IP address types
+func findIpAddr(s *RemoteCertSource, data *sqladmin.DatabaseInstance, instance string) (ipAddrInUse string, err error) {
```

Comment:
Now that this is extracted into a function, it'd be nice to add tests for it ;)

Also, IMO you may as well make this a function that is part of the RemoteCertSource struct.

Reply:
Sure. Will add the test later.